src/Aspire.Hosting/Aspire.Hosting.csproj

```diff
@@ -33,13 +33,12 @@
     <Compile Include="$(SharedDir)LoggingHelpers.cs" Link="Utils\LoggingHelpers.cs" />
     <Compile Include="$(SharedDir)StringUtils.cs" Link="Utils\StringUtils.cs" />
     <Compile Include="$(SharedDir)SchemaUtils.cs" Link="Utils\SchemaUtils.cs" />
-    <Compile Include="$(SharedDir)LaunchSettings.cs" Link="LaunchSettings.cs" />
-    <Compile Include="$(SharedDir)LaunchProfile.cs" Link="LaunchProfile.cs" />
-    <Compile Include="$(SharedDir)LaunchSettingsSerializerContext.cs" Link="LaunchSettingsSerializerContext.cs" />
     <Compile Include="$(SharedDir)SecretsStore.cs" Link="Utils\SecretsStore.cs" />
     <Compile Include="$(SharedDir)ConsoleLogs\LogEntries.cs" Link="Utils\ConsoleLogs\LogEntries.cs" />
     <Compile Include="$(SharedDir)ConsoleLogs\LogEntry.cs" Link="Utils\ConsoleLogs\LogEntry.cs" />
     <Compile Include="$(SharedDir)ConsoleLogs\TimestampParser.cs" Link="Utils\ConsoleLogs\TimestampParser.cs" />
+    <Compile Include="$(SharedDir)CommandLineArgsParser.cs" Link="Utils\CommandLineArgsParser.cs" />
+    <Compile Include="$(SharedDir)LaunchProfiles\*.cs" />
```

Comment:
This doesn't seem right. There is code in `ApplicationExecutor` that relies on `LaunchProfileAnnoation`. This means we'll have two different `LaunchProfileAnnotation` types - the the on that `Aspire.Hosting` understands and the one that `Aspire.Hosting.Azure.Functions` understands.

Reply:
This annotation seems to only store the name of the launch profile to be used. Is there a particular concern for the divergence, especially given they reference the same shared source for this implementation?

Reply:
Right now I think it is OK because we aren't using the LaunchProfileAnnotation in a way that the divergence will hurt us, but I'm thinking of how much of a pain to debug this might be in the future when we are looking for the launchprofileannotation type and its not showing up :)